<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientPool</class>
 <widget class="QDockWidget" name="clientPool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="baseSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="features">
   <set>QDockWidget::NoDockWidgetFeatures</set>
  </property>
  <property name="windowTitle">
   <string>ClientPool</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>581</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>参数配置</string>
    </property>
    <widget class="QPushButton" name="pushButton_saveConfig">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>20</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存配置</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>120</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>开始仿真</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>通信模式：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_devNum">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>20</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_comm_model">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>TCP</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>LORA</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>设备数量：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>设备名前缀：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_startDevId">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>461</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>TCP服务器参数</string>
     </property>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>IP:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_serverIP">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>113</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>端口：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_serverPort">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>461</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>LORA参数</string>
     </property>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>波特率：</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>自身地址：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Addr">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>20</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>信道：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_channel">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>20</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_BaudRate">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>20</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="pushButton_loraDebug">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>70</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>LORA调试</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>350</y>
      <width>251</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>仿真控制</string>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>72</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>异常状态：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_condition">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>91</width>
       <height>25</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>电机堵转</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>低电量</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>离树报警</string>
      </property>
     </item>
    </widget>
    <widget class="QPushButton" name="pushButton_send">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>131</height>
      </rect>
     </property>
     <property name="title">
      <string>系统参数</string>
     </property>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>72</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>睡眠时间：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_sleepTime">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>60</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>72</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>收集时间：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_collectTime">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>60</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>72</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>割胶时间：</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_waitTime">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>90</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>10</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_status">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>60</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>获取状态</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_resetParam">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>71</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>重置参数</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>321</width>
      <height>351</height>
     </rect>
    </property>
    <property name="title">
     <string>输出日志</string>
    </property>
    <widget class="QTextEdit" name="textEdit_log">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>301</width>
       <height>321</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>190</y>
      <width>251</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>设备列表</string>
    </property>
    <widget class="QListWidget" name="listWidget_dev">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>231</width>
       <height>121</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>设备1</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>设备6</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>设备2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>设备3</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>设备4</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>设备5</string>
      </property>
     </item>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
